internal/sign/helper.go

```diff
@@ -16,29 +17,45 @@ func NewSignerHelper() Helper {
 	return &helper{}
 }
 
-func (m *helper) GetRelevantSign(modSpec kmmv1beta1.ModuleSpec, km kmmv1beta1.KernelMapping) *kmmv1beta1.Sign {
-	if modSpec.ModuleLoader.Container.Sign == nil {
-		// km.Sign cannot be nil in case mod.Sign is nil, checked above
-		return km.Sign.DeepCopy()
-	}
+func (m *helper) replaceInParam(macros map[string]string, params ...string) []string {
 
-	if km.Sign == nil {
-		return modSpec.ModuleLoader.Container.Sign.DeepCopy()
+	var replacedStrings []string
+	for k, v := range macros {
+		m := "${" + k + "}"
```

Comment:
nit.
```suggestion
		template := "${" + k + "}"
```

Reply:
as always you are correct, good nit, done!